source/index.html.md

```diff
@@ -107,19 +113,23 @@ HIVE_SDK(
 }
 ```
 
-This call will take the provided user data, and use it to authenticate the current user and add them to your brand's contact list in Hive.
+This call will take the provided user data, and use it to "authenticate" the current user and add them to your brand's contact list in Hive.
 
 The <code>email</code> field is the only required field, although passing along as much data as possible is recommended. For instance, including <code>firstName</code> will help to generate a gender for your contacts in Hive. <code>location</code> should be a string with as much location granularity as possible (we'll take care of geocoding, etc. on our own).
 
+If you're including a user's phone number, it is recommended to provide phone numbers in E.164 or RFC3966 format to ensure that phone numbers are properly saved, . In cases where Hive cannot resolve a poorly formatted phone number, a default country code of `+1` (United States/Canada) will be assumed.
```

Comment:
```suggestion
If you're including a user's phone number, it is recommended to provide phone numbers in E.164 or RFC3966 format to ensure that phone numbers are properly saved. In cases where Hive cannot resolve a poorly formatted phone number, a default country code of `+1` (United States/Canada) will be assumed.
```

Reply:
add a note/warning about the implications of assuming a +1 that may result in a user not getting saved at all?

Reply:
also side note: maybe we should enforce E.164 formatting and not make any assumptions at least with the sdk?